evalml/tests/data_checks_tests/test_datetime_nan_data_check.py

```diff
@@ -1,6 +1,5 @@
 import numpy as np
 import pandas as pd
-import woodwork as ww
```

Comment:
Question, but why do we no longer need to import woodwork, even if we're using `ww.init()` and other ww methods?

Reply:
Great observation! This is something I hadn't noticed actually. 

In order to use the accessor, woodwork needs to be present in the namespace but it doesn't need to be imported in the same file that you use it. Since our  `evaml/__init__.py` file imports `ww` by way of importing our submodules, `ww` is imported when our tests run so we can use the accessor. 

I suspect I originally kept the import but then deleted it because of lint since it's not "needed".

Reply:
Oh that's interesting. Thanks for the clarification!